tests/tokens.spec.ts

```diff
@@ -81,7 +81,8 @@ describe("Test Tokens", () => {
       "0xB97048628DB6B661D4C2aA833e95Dbe1A905B280"
     ) as Asset;
     expect(payAsset.name).toEqual("PAY");
-    expect(payAsset.decimals).toEqual(18);
+    // @ts-ignore: if payAsset.contract is undefined, the test will fail as expected
+    expect(payAsset.contract.decimals).toEqual(18);
```

Comment:
:upside_down_face: Maybe better, maybe worse.

```suggestion
    expect(payAsset).toBeDefined();
    expect(payAsset.contract).toBeDefined();
    expect(payAsset!.contract!.decimals).toEqual(18);
```

Reply:
🤯